packages/davinci-client/src/lib/node.slice.test.ts

```diff
@@ -88,9 +88,6 @@ describe('The node slice reducers', () => {
       },
       httpStatus: 400,
       server: {
-        id: 'cqm26fxq1v',
```

Comment:
I think this test, should probably be better written, so I will update this after initial feedback.

This test probably needs to test given a state of a continue node, that the error node updates properly.